runelite-client/src/main/java/net/runelite/client/plugins/performance/PerformanceOverlay.java

```diff
@@ -45,25 +47,26 @@
  * This locks "FPS:" into one position (the far top right corner of the canvas),
  * along with a locked position for the FPS value.
  */
-public class FpsOverlay extends Overlay
+public class PerformanceOverlay extends Overlay
 {
 	private static final int Y_OFFSET = 1;
-	private static final int VALUE_X_OFFSET = 1;
-	private static final String FPS_STRING = " FPS";
 
 	// Local dependencies
-	private final FpsConfig config;
+	private final PerformanceConfig config;
 	private final Client client;
 
 	// Often changing values
 	private boolean isFocused = true;
 
 	@Inject
-	private FpsOverlay(FpsConfig config, Client client)
+	private PerformancePlugin plugin;
+
+	@Inject
+	private PerformanceOverlay(PerformanceConfig config, Client client)
 	{
 		this.config = config;
 		this.client = client;
-		setLayer(OverlayLayer.ABOVE_WIDGETS);
+		setLayer(OverlayLayer.ALWAYS_ON_TOP);
```

Comment:
Why?

Reply:
I took this from the original PR, I'll revert it.